DataGrid: add Custom keyboard navigation topic (#5573)
DevExpress/devextreme-documentation: concepts/05 UI Components/DataGrid/70 Custom Keyboard Navigation/05 Change the Default Focused Cell.md

> To specify which cell should be focused next when a user navigates through DataGrid, use the [onFocusedCellChanging](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onFocusedCellChanging) function. The code below uses `newColumnIndex` to set the column index of the next focused cell.

Reviewer: I believe we can say the same with fewer words and in a more focused manner.

> use the X function

How can I use a function? I can call/invoke it. If we can add a bit more clarity, we should.

```suggestion
To specify cell focus order when a user navigates through DataGrid, call the [onFocusedCellChanging](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onFocusedCellChanging) function. The code below uses `newColumnIndex` to set the column index of the next focused cell.
```

---
##### jQuery

    <!-- tab: index.js -->
    $("#dataGridContainer").dxDataGrid({
        // ...
        onFocusedCellChanging: function (e) {

>             if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell will be in the 'myField' column

Reviewer: If we are checking this condition right now, that means the thing we describe also happens in the present.

```suggestion
            if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell is in the 'myField' column
```

>                 e.newColumnIndex++; // If so, proceed to the next column

Reviewer: > If so,

We already declared the checked condition in the previous line, didn't we?

```suggestion
                e.newColumnIndex++; // Navigates to the next column
```

Please also keep consistency in mind. In the previous line you said what it does -> **Checks** if ...

Therefore, we should address all other code lines in the same manner.

            }
        }
    });

##### Angular

    <!-- tab: app.component.html -->
    <dx-data-grid ...
        (onFocusedCellChanging)="onFocusedCellChanging($event)">
    </dx-data-grid>

    <!-- tab: app.component.ts -->
    import { Component } from '@angular/core';
    
    @Component({
        selector: 'app-root',
        templateUrl: './app.component.html',
        styleUrls: ['./app.component.css']
    })
    
    export class AppComponent {
        onFocusedCellChanging(e) {

>             if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell will be in the 'myField' column

Reviewer: Likewise.

```suggestion
            if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell is in the 'myField' column
```

>                 e.newColumnIndex++; // If so, proceed to the next column

Reviewer: Likewise.

```suggestion
                e.newColumnIndex++; // Navigates to the next column
```

            }
        }
    }

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxDataGrid ...
            @focused-cell-changing="onFocusedCellChanging">
        </DxDataGrid>
    </template>
    
    <script>
    import 'devextreme/dist/css/dx.light.css';
    import DxDataGrid from 'devextreme-vue/data-grid';
    
    export default {
        components: {
            DxDataGrid
        },
        // ...
        methods: {
            onFocusedCellChanging(e) {

>                 if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell will be in the 'myField' column

Reviewer: ```suggestion
                if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell is in the 'myField' column
```

>                     e.newColumnIndex++; // If so, proceed to the next column

Reviewer: ```suggestion
                    e.newColumnIndex++; // Navigates to the next column
```

                }
            }
        }
    }
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';
    import 'devextreme/dist/css/dx.light.css';
    import DataGrid from 'devextreme-react/data-grid';

    const onFocusedCellChanging = (e) => {

>         if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell will be in the 'myField' column

Reviewer: Likewise.

```suggestion
        if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell is in the 'myField' column
```

>             e.newColumnIndex++; // If so, proceed to the next column

Reviewer: Likewise.

```suggestion
            e.newColumnIndex++; // Navigates to the next column
```
